js/ui/shellDBus.js

```diff
@@ -607,7 +615,6 @@ const AppLauncher = new Lang.Class({
 
         if (appName == 'eos-app-store') {
             Main.appStore.show(timestamp, true);
-            Main.appStore.appLaunched = true;
```

Comment:
Oh, I see that you undo the change in this commit. Please squash it instead with the previous commit, so that none of the two changes this behavior.